examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -15,181 +14,190 @@ class CupertinoAlertDemo extends StatefulWidget {
+                  CupertinoButton.filled(
+                    child: const Text('Alert'),
+                    onPressed: () {
+                      showDemoDialog(
+                        context: context,
+                        child: CupertinoAlertDialog(
+                          title: const Text('Discard draft?'),
+                          actions: <Widget>[
+                            CupertinoDialogAction(
+                              child: const Text('Discard'),
+                              isDestructiveAction: true,
+                              onPressed: () {
+                                Navigator.pop(context, 'Discard');
+                              },
+                            ),
+                            CupertinoDialogAction(
+                              child: const Text('Cancel'),
+                              isDefaultAction: true,
+                              onPressed: () {
+                                Navigator.pop(context, 'Cancel');
+                              },
+                            ),
+                          ],
+                        ),
+                      );
+                    },
+                  ),
+                  const Padding(padding: EdgeInsets.all(8.0)),
+                  CupertinoButton.filled(
+                    child: const Text('Alert with Title'),
+                    padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
                     onPressed: () {
-                      Navigator.pop(context, 'Cancel');
+                      showDemoDialog(
+                        context: context,
+                        child: CupertinoAlertDialog(
+                          title: const Text('Allow "Maps" to access your location while you are using the app?'),
+                          content: const Text('Your current location will be displayed on the map and used '
+                            'for directions, nearby search results, and estimated travel times.'),
+                          actions: <Widget>[
+                            CupertinoDialogAction(
+                              child: const Text('Don\'t Allow'),
+                              onPressed: () {
+                                Navigator.pop(context, 'Disallow');
+                              },
+                            ),
+                            CupertinoDialogAction(
+                              child: const Text('Allow'),
+                              onPressed: () {
+                                Navigator.pop(context, 'Allow');
+                              },
+                            ),
+                          ],
+                        ),
+                      );
                     },
-                  )
+                  ),
+                  const Padding(padding: EdgeInsets.all(8.0)),
+                  CupertinoButton.filled(
+                    child: const Text('Alert with Buttons'),
```

Comment:
It doesn't look like this dialog actually has buttons.  Are those missing?

Reply:
Actually, it looks like its a CupertinoDessertDialog so I assume that implicitly contains buttons.